cloud_governance/common/clouds/ibm/developer_tools/schematic_operations.py

```
@@ -34,22 +34,31 @@ def get_workspaces(self):
-            if geography_code not in resources_list:
-                resources_list[geography_code] = self.get_workspaces()
+
+        for region in known_regions:
+            try:
+                self.set_service_url(region)
+                workspaces = self.get_workspaces()
+                if workspaces:
+                    resources_list[region] = workspaces
+            except Exception:
+                # Skip regions where schematics is not available or no access
+                continue
```

Ruff (0.15.18)

[error] `try`-`except`-`continue` detected, consider logging the exception

(S112)